deployment/configCmd.py

```diff
@@ -0,0 +1,209 @@
+import os
+import argparse
+from deployment.confStorage.download import download_configuration
+from deployment.confStorage.synchronization import synchronization
+from deployment.confStorage.external_version_control.external_config import uploading_external_config
+from deployment.confStorage.get_cluster_id import get_cluster_id
+
+
+import logging
+import logging.config
+import importlib
+from paiLibrary.common import file_handler
+from paiLibrary.common import template_handler
+from k8sPaiLibrary.maintainlib import common as pai_common
+
+
+logger = logging.getLogger(__name__)
+
+
+def generate_configuration(quick_start_config_file, configuration_directory, force):
+    """Automatically generate the following configuration files from a quick-start file:
+        * Machine-level configurations: laylout.yaml
+        * Kubernetes-level configurations I: kubernetes-configuration.yaml
+        * Kubernetes-level configurations II: k8s-role-definition.yaml
+        * Service-level configurations: service-configuration.yaml
+    """
+    quick_start_config_raw = file_handler.load_yaml_config(quick_start_config_file)
```

Comment:
put it in a single module?

Reply:
I avoid to use any module, class, object attributes when there is no need.
Instead, using raw function (group into files), local variable.